bot.py

```diff
@@ -213,37 +251,68 @@ def check(bot):
             return
     filter_list = ['ramen', 'fries', 'ah', 'sandwich']
 
-    voted = get_slack_names(bot, [reaction for reaction in reactions['message']['reactions']
-                                  if reaction['name'] in filter_list])
+    voted = get_slack_names(
+        bot,
+        [
+            reaction
+            for reaction in reactions['message']['reactions']
+            if reaction['name'] in filter_list
+        ]
+    )
+
     lowest = wbw_get_lowest_member(voted)
     try:
-        votes = [(reaction['name'], reaction['count']) for reaction in reactions['message']['reactions']
-                 if reaction['name'] in filter_list]
+        votes = [
+            (reaction['name'], reaction['count'])
+            for reaction in reactions['message']['reactions']
+            if reaction['name'] in filter_list
+        ]
+
         # Filter out our own reactions
         votes = filter(lambda x: x[1] != 1, votes)
-        # The multiple max allows us to make a random choice when the vote is tied
+
+        # The multiple max allows us to make a random choice when the vote
+        # is tied
         try:
-            choice = random.choice(multiple_max(votes, key=lambda x: x[1]))[0]
+
+            highest_vote = max(votes, key=lambda i: i[1])[1]
+
+            choice = random.choice(
```

Comment:
You should add a comment here because you replaced a fully explained function with a one-liner